pkg/liquidity-mining/contracts/gauges/ChildChainGauge.vy

```diff
@@ -701,12 +690,12 @@ def authorizer_adaptor() -> address:
 
 
 @external
-def initialize(_lp_token: address):
+def initialize(_lp_token: address, _voting_escrow: address, _version: String[128]):
     assert self.lp_token == ZERO_ADDRESS  # dev: already initialzed
 
     self.lp_token = _lp_token
-
-    self.voting_escrow = Factory(msg.sender).voting_escrow()
+    self.voting_escrow = _voting_escrow
```

Comment:
Now that you don't store the factory any more, you can make this immutable and set it in the constructor instead of here